rtb-house/rtb-house-htb.js

```diff
@@ -129,11 +129,41 @@ function RTBHouseHtb(configs) {
          */
 
         /* ---------------------- PUT CODE HERE ------------------------------------ */
-        var queryObj = {};
         var callbackId = System.generateUniqueId();
+        var queryObj = {
+            id: callbackId,
+            imp: [],
+            site: {
+                publisher: {
+                    id: configs.publisherId
+                },
+                page: Browser.getPathname(),
+                name: Browser.getHostname()
+            },
+            cur: ['USD'],
+            test: 0,
+            source: {
```

Comment:
po co ten zamarkowany, pusty source? On jest jakos potrzebny?

Reply:
Był opcjonalny, to go wrzuciłem pustego; ale faktycznie nic z nim później nie robię - wyrzucam.